packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -187,11 +187,10 @@ export const FeedContainer = ({
       )}
     >
       {isV1Search && shouldUseFeedLayoutV1 && (
-        <img
-          className="absolute left-0 top-0 w-full -translate-y-1/2"
-          src={cloudinary.feed.bg.layoutV1}
-          alt="Gradient background"
-        />
+        <div className="absolute -top-24 left-1/2 flex -translate-x-1/2 flex-row justify-center">
+          <span className="z-1 size-40 translate-x-6 rounded-full bg-theme-color-cabbage blur-[5rem]" />
```

Comment:
Using translate instead of margin seems unnecessary to me. We can also get rid of the z-index here by swapping the order of the elements, e.g. ellipse first and circle second in the DOM.

Reply:
Yeah, we can probably remove z-index by doing flow reverse then swap the elements around.

With regards to the translate vs margin. I think it is more appropriate to use translate when fixing the actual placement of the element. Margins, I believe, are used for spacing, just probably usable when moving elements as well. Wdyt?